### PR TITLE
/uuid and /name cmds + new util

### PR DESCRIPTION
**Pull request by MrSam7K**
/uuid and /name commands
new utility to turn trimmed uuid into uuid with dashes
image

### CODE REVIEW

File: src/main/java/dev/dfonline/codeclient/command/impl/CommandName.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                                if(localUsername.equals(fullName)) {
+                                    Utility.sendMessage(Text.translatable("codeclient.command.name.own"), ChatType.SUCCESS);
+                                    return 0;
+                                } else {
+                                    Text message = Text.empty()
+                                            .append(Text.translatable("codeclient.command.name", uuid))
+                                            .append(Text.literal(fullName).formatted(Formatting.AQUA, Formatting.UNDERLINE))
+                                            .append(Text.literal("!").formatted(Formatting.RESET))
+                                            .fillStyle(Style.EMPTY
+                                                    .withHoverEvent(new HoverEvent(HoverEvent.Action.SHOW_TEXT, Text.literal("Click to copy to clipboard")))
```
Comment (RedVortexDev) [inline src/main/java/dev/dfonline/codeclient/command/impl/CommandName.java:68]: Please replace with a Text.translatable

File: src/main/resources/assets/codeclient/lang/en_us.json [hunk 2]
```diff
@@ -278,5 +280,11 @@
...
   "codeclient.command.dfgive.count_below_min": "Minimum stack size is 1",
   "codeclient.command.dfgive.clipboard_unavailable": "Unable to access the clipboard",
-  "codeclient.command.dfgive.clipboard_invalid": "Please copy a valid give command to your clipboard"
+  "codeclient.command.dfgive.clipboard_invalid": "Please copy a valid give command to your clipboard",
+  "codeclient.command.uuid.not_found": "Player %s was not found",
+  "codeclient.command.uuid": "UUID of %s is ",
+  "codeclient.command.uuid.own": "Your UUID is ",
+  "codeclient.command.name.not_found": "UUID %s was not found! Please check if you misspelled it and try again",
+  "codeclient.command.name": "Name of %s is ",
+  "codeclient.command.name.own": "The UUID is yours! What a surprise!"
```
Comment (RedVortexDev) [inline src/main/resources/assets/codeclient/lang/en_us.json:289]: uhh

**Review by RedVortexDev — CHANGES_REQUESTED** [review]
haven't taken a deep look yet, but I did notice some things

File: src/main/resources/assets/codeclient/lang/en_us.json (hunk 2 above)
Comment (MrSam7K) [reply src/main/resources/assets/codeclient/lang/en_us.json:289]: I did not know how else I could keep it translatable but underline uuid/name in the message

File: src/main/java/dev/dfonline/codeclient/command/impl/CommandName.java (hunk 1 above)
Comment (MrSam7K) [reply src/main/java/dev/dfonline/codeclient/command/impl/CommandName.java:68]: Oh yeah I forgot to do that with hover

**Comment by MrSam7K** [comment]
here, i actually did not know you could do that

File: src/main/resources/assets/codeclient/lang/en_us.json (hunk 2 above)
Comment (RedVortexDev) [reply src/main/resources/assets/codeclient/lang/en_us.json:290]: i was talking about the name.own one

Comment (MrSam7K) [reply src/main/resources/assets/codeclient/lang/en_us.json:290]: yeah so? its a funny little easter egg B)

**Comment by MrSam7K** [comment]
any updates?

**Review by GeorgeRNG — APPROVED** [review]

**Pull request merged**